bin/yarn

```diff
@@ -2,5 +2,9 @@
 
 set -e
 
-IMAGE_NAME=${IMAGE_NAME:-codeclimate/codeclimate-eslint}
-docker run --rm --volume "$PWD:/usr/src/app" "$IMAGE_NAME" sh -c "cd /usr/src/app && yarn $*"
+docker run \
+  --rm \
+  --user "$(id -u):$(id -g)" \
```

Comment:
This may not work as desired for Mac users: I know the Docker for Mac VM runs as root, and I think I ran into some issues with user perms and mounts working with that at one point. 

Come to think of it, that might have been for stuff within a Dockerfile, so this might work fine. Might want to have a Mac user confirm before merging, though.